assets/data-port/import/done/done-page.js

```diff
@@ -0,0 +1,85 @@
+import { __ } from '@wordpress/i18n';
+import { Section } from '@woocommerce/components';
+import { useEffect } from '@wordpress/element';
+import { formatString } from '../../../setup-wizard/helpers/format-string';
```

Comment:
We might want to consider moving this to a shared namespace

Reply:
Good catch, 5348b32